docs: adding initial content for v2 (#290)
klarna/electron-redux: docs/docs/introduction/getting-started.md

---
id: getting-started
title: Getting Started with electron-redux
description: 'Introduction > Getting Started: Resources to get started with electron-redux'
hide_title: false
---

> Electron-Redux is an Redux Store Enhancer that helps you loosely synchronize the redux stores in multiple electron processes.

Reviewer: ```suggestion
electron-redux is a Redux Store Enhancer that helps you loosely synchronize the redux stores in multiple electron processes.
```

Reviewer: Related - what do you mean by "loosely synchronise"?

Author: @hardchor Maybe the wording isn't right, but I want to emphasize the fact that the stores in main/renderer are not actually synchronized out of the box and there is no mechanism with the library that will actually provide that - we don't sync the states, we just make sure the actions are accessible from both places. Any clues on different more self-explanatory naming?

> When working with Electron, using Redux poses couple of problems, since main and renderer processes are isolated and IPC is a single way of communication between them. Electron-Redux, lets you to register all your Redux stores in the main & renderer process, and enable cross-process action dispatching & loose store synchronization.

Reviewer: ```suggestion
When working with Electron, using Redux poses a couple of problems, since main and renderer processes are isolated and IPC is the only way of communication between them. Electron-redux, allows you to register all your Redux stores in the main & renderer process, and enable cross-process action dispatching & loose store synchronization.
```

# Installation

> Electron-Redux is available as a package on NPM for use with a module bundler:

Reviewer: ```suggestion
`electron-redux` is available as a package on NPM for use with a module bundler:
```

```sh
# Yarn
yarn add electron-redux@alpha

# NPM
npm install electron-redux@alpha
```

# Configuration

> electron-redux comes as a [Redux StoreEnhancer](https://redux.js.org/understanding/thinking-in-redux/glossary#store-enhancer). To initialize your stores, you just need to decorate them in the `main` and `renderer` processes of electron with their respective enhancers:

Reviewer: ```suggestion
electron-redux comes as a [Redux store enhancer](https://redux.js.org/understanding/thinking-in-redux/glossary#store-enhancer). To initialize your stores, you just need to decorate them in the `main` and `renderer` processes of electron with their respective enhancers:
```
